### DISCUSSION TITLE
changelog

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#77] [Announcements]
# changelog

粥里有勺糖的博客主题，基于 vitepress 实现

https://theme.sugarat.top/changelog.html

**Comment by pejaz** [comment]
大佬，你博客是部署在哪里的？想找个省钱国内体验又友好的方案

**Reply by ATQQ** [reply]
我用的腾讯云服务器。买的国内的机子

**Reply by ATQQ** [reply]
买活动机器，可以各厂商看一下

比如这种 (最近看到的) https://www.volcengine.com/activity/promotion-juejin?utm_campaign=juejin_navbar&utm_content=juejin_dev_amd&utm_medium=juejin&utm_source=external_cooperation&utm_term=juejin_dev_amd

**Reply by pejaz** [reply]
我还以为你直接部署到cos上。买云主器这种继续到期要迁移，不会麻烦么

**Reply by ATQQ** [reply]
各有利弊吧。看自己。全用cdn 万一被攻击 一晚上 一套房都打没了

服务器迁移其实还好 ，可以买时间长点

**Reply by pejaz** [reply]
😂cos能不能设置流量超出就停止访问。宁愿服务挂了也不能让攻击到破产

**Comment by wjp980108** [comment]
佬，请问支持在文章页内设置作者跳转外链吗？

**Comment by pejaz** [comment]
我最近更新完新版本后发现文章内 article 部分不显示了。是不是新版本和旧版本哪里设置问题？方便看下吗？博客链接：https://peigo.top/

**Reply by pejaz** [reply]
忽略，找到原因了，文章内没有加入一级标题导致的（奇怪的是为什么不会自动读取 title 作为一级标题？我记得以前是会的）。

**Reply by ATQQ** [reply]
回头我看下👌🏻

**Reply by pejaz** [reply]
我想可能之前是用的 vuepress 转过来的。不过这个功能我觉得还挺方便的。并且在 vscode 中使用 markdown-lint 会把 yaml 里面的 title 也识别为一个 h1 ，再写一个相同的 h1 标题会提示重复。后续方便的话可以看一下。当然注意身体～

**Reply by pejaz** [reply]
> 回头我看下👌🏻

这里我简单写了个 markdown 插件实现了，[vitepress-plugin-frontmatter](https://www.npmjs.com/package/vitepress-plugin-frontmatter)，有需要可以自取👀
